Units/aiunits/eaiturret.lua

```diff
@@ -57,8 +57,8 @@ local unitDef                    = {
 	leaveTracks                  = false,
 	maxDamage                    = maxDamage,
 	maxSlope                     = 26,
-	maxVelocity                  = 0.001,
-	maxReverseVelocity           = 0.001,
+	maxVelocity                  = 1,

```

Comment:
Why is this building moving? It would be better to use an actual thud instead of something with a platform under it. You could alter the thud to be super slow.

Switching out the model and script lines for the ones in the actual thud would likely do it.